bbs/admin/inifiles/net.ini

```diff
@@ -0,0 +1,10 @@
+;=============================================================================
+;
+;                Network Version 5.3.0 Initialization File
+;
+;              Lines begining with a semi-colon are ignored
+;=============================================================================
+;
```

Comment:
just FYI - blank lines are ignored too (in my ini parser, not in the original one), so
we don't need to have empty lines contain a ;